demos/object_detection_demo/cpp/README.md

```diff
@@ -136,6 +136,7 @@ python3 <omz_dir>/tools/downloader/converter.py --list models.lst
   - yolo-v2-tiny-vehicle-detection-0001
   - yolo-v4-tf
   - yolo-v4-tiny-tf
+  - yolof
```

Comment:
pls also add `yolof` to description of `-at` option,

-at "<type>"              Required. Architecture type: centernet, faceboxes, retinaface, retinaface-pytorch, ssd or yolo

Reply:
In cpp demo, YOLOF model using -at `yolo`.

Reply:
why it is differs from python? in general it is better to maintain unified cmd line options across similar demos, to not confuse customers

Reply:
ok, C++ model api should catch up design of python model api